### DISCUSSION TITLE
Seed for RF

### DISCUSSION THREAD

**Discussion by lskatz** [kogalur/randomForestSRC#412] [Q&A] [answered]
When making a random forest model, does it save the seed that was used inside the object itself? I wanted to save the model to an rds file and name the file after the seed that was given. And you can specify the seed when you call rfsrc if you use e.g., `seed = 42`, right? Thank you for your help!

**Comment by kogalur** [comment, marked as answer]
Most of the function calls in the package accept a seed (negative integer) to allow reproducibility.  If you use a positive integer it will just be converted to a negative integer internally.  You will always be able to restore the model with the seed.  There are certain conditions in which statistics like VIMP will be subject to Monte Carlo effects.  There's a long discussion in this forum on that topic that you can search for if you need the subtleties going forward.

**Reply by lskatz** [reply]
Thank you!
